hack/generator/pkg/codegen/pipeline/testdata/TestInjectPropertyAssignmentTests/microsoft.person-v20211231_test.golden

```diff
@@ -0,0 +1,422 @@
+// Code generated by azure-service-operator-codegen. DO NOT EDIT.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package v20211231
+
+import (
+	"encoding/json"
+	"github.com/Azure/azure-service-operator/testing/microsoft.person/v20211231storage"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/kr/pretty"
+	"github.com/kylelemons/godebug/diff"
+	"github.com/leanovate/gopter"
+	"github.com/leanovate/gopter/gen"
+	"github.com/leanovate/gopter/prop"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func Test_Person_WhenPropertiesConverted_RoundTripsWithoutLoss(t *testing.T) {
+	parameters := gopter.DefaultTestParameters()
+	parameters.MaxSize = 10
+	properties := gopter.NewProperties(parameters)
+	properties.Property(
+		"Round trip from Person to Person via AssignPropertiesToPerson & AssignPropertiesFromPerson returns original",
+		prop.ForAll(RunPropertyAssignmentTestForPerson, PersonGenerator()))
+	properties.TestingRun(t, gopter.NewFormatedReporter(false, 240, os.Stdout))
```

Comment:
Every time I see "formated" my eye twitches. I'm guessing you feel the same way.

Reply:
Yup. 

![gif](https://media.giphy.com/media/KymorXwDdmvw4/giphy.gif)